packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -674,7 +706,13 @@ export class AtomicSearchBox {
         class={`flex w-full z-10 absolute left-0 top-full rounded-md bg-background border border-neutral ${
           this.showSuggestions ? '' : 'hidden'
         }`}
-        role="listbox"
+        role="application"
```

Comment:
Could the role stay a simple listbox if there's no double list?

Reply:
Would perhaps add better support for Mac for customers that don't use instant results (probably most customers)

Reply:
perhaps, but I prefer to keep the behaviour consistent, since both behave pretty differently and it would be like testing two components.

Reply:
Valid point & the end result will be pretty much the same to the screen reader user

Reply:
Hum... yeah I would tend to agree with Jean-Felix on this one.

I'd say realistically speaking, vast majority of clients will just be using normal query suggestions, and instant results will be the exception.

So it's like we will offer something less robust with role="application" (that's my understanding that it's less robust, correct me if I'm wrong), in order to support something that most client will not use...

Reply:
Yes, it's a little bit less robust on macOS with VoiceOver. Apart from that, it seems pretty robust.

I would prefer to wait and see if clients report this as an accessibility issue.